lib/json/store.nit

```diff
@@ -111,7 +111,7 @@ class JsonStore
 
 	# Is there data are stored under `key`.
 	fun has_key(key: String): Bool do
-		return (store_dir / "{key}.json").file_exists
+		return ("{store_dir}/{key}.json".simplify_path).file_exists
```

Comment:
The problem with the `/` operator is that it doesn't like the directory starting with `.` like in `.github_data`.

~~~nit
print ".github_data" / "foo" # prints "/foo"
~~~

So I use the literal `/` to make the path then use `simplify_path` to remove the useless `///` from the path.